### PR TITLE
V2.16 changelog update

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/home/changelog/hotfixes-2-16.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+### Release content
+
+* <!-- RUN-12664 - [Control-plane] Implement get workload/pods API -->Implemented a new Workloads API to support the *Workloads* feature.
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-16346 | Fixed an issue where the *Jobs* and *Deployments* pages were not displayed for trial users. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:39]: Please remove. Not customer facing as trial is not available yet

File: docs/home/changelog/hotfixes-2-16.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+| RUN-16070 | Fixed an issue where missing metrics caused the *Nodepools* table to appear empty. |
+
+## Version 2.16.14
+
+### Release content
+
+*<!-- RUN-16108 | Reduce the frequency of fetching metrics from 10 sec to 30 sec -->Improved overall performance by slowing down metrics updates from 10 seconds to 30 seconds.
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:46]: @gal-revach , is this hotfix specific? Should it be connected to this specific hotfix on this specific version?

File: docs/home/changelog/hotfixes-2-16.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-16255 | Fixed an issue in the *Analytics* dashboard where some node panels were displaying incorrect data. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:52]: It's not the nodepools that showed incorrect data. Look at the bug - Natasha listed the specific 2 widgets. Nodepools are the cause.
This one needs to be rewritten.

File: docs/home/changelog/hotfixes-2-16.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+| RUN-16035 | Fixed an issue in the *Workloads* table where completed pods continue to be counted in the requested resources column. |
+
+## Version 2.16.12
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-16110 | Fixed an issue where creating a training workload (Standard or Distributed) with a new PVC or Volume, resulted in the *Workloads* table showing the workload in the Unknown/Pending status. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:61]: we do not use the term "Standard" for this type of workloads. Please use the term we use.

File: docs/home/changelog/hotfixes-2-16.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+| RUN-16086 | Fixed an issue in airgapped environments where incorrect installation commands were shown. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:62]: According to the bug it happened in 2.13->2.15 upgrade. I understand the code was merged to 2.16.
Is it related to 2.16? to upgrade only or any installation? The description as you wrote is not clear enough

File: docs/home/changelog/hotfixes-2-16.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+## Version 2.16.11
+
+| Internal ID | Description |
+|--|--|
+| RUN-16102 | Fixed an issue where browser pages behaved strangely for users who logged in and did not have the correct permissions. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:68]: Please remove

File: docs/home/changelog/hotfixes-2-16.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+| RUN-16068 | Fixed an issue in the UI where the *Projects* selection pane is not working. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:69]: The description is not correct. It is in the Departments screen and the Projects pop-up is empty. It is all in the bug.

Anyway, let's remove it altogether.

File: docs/home/changelog/hotfixes-2-16.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+### Release content
+
+* <!-- | RUN-12664 | [Control-plane] Implement get workload/pods API | -->Added an API endpoint that retrieves data from a workloads's pod.
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-15709 | Fixed an issue where internal system audits were exposed to the event history in the API and the UI. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:91]: Please remove

File: docs/home/changelog/hotfixes-2-16.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+| RUN-15706 | Fixed an issue in the *Workspace* table, where the copy and edit buttons did not work. |
```
Comment (omerbenedictrunai) [inline docs/home/changelog/hotfixes-2-16.md:92]: Not clear exactly where and when. Please either update with more details or remove.

**Review by omerbenedictrunai — COMMENTED** [review]
Added a few comments

File: docs/home/changelog/hotfixes-2-16.md (hunk 2 above)
Comment (gal-revach) [reply docs/home/changelog/hotfixes-2-16.md:46]: @omerbenedictrunai @jasonnovichRunAI yes.
This was added in the following versions:
2.13.43
2.16.16

File: docs/home/changelog/hotfixes-2-16.md (hunk 4 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-16.md:61]: @omerbenedictrunai I changed the terminology to match this page https://docs.run.ai/v2.16/Researcher/user-interface/trainings/?h=training+workload#adding-trainings see #3.

File: docs/home/changelog/hotfixes-2-16.md (hunk 5 above)
Comment (jasonnovichRunAI) [reply docs/home/changelog/hotfixes-2-16.md:62]: @omerbenedictrunai I would like to move this the 2.15 changelog. WDYT?
```suggestion
| RUN-16086 | Fixed an issue in airgapped environments where incorrect installation commands were shown when upgrading to V2.15. |
```

**Comment by jasonnovichRunAI** [comment]
@omerbenedictrunai Ready for last review.

**Pull request merged**